lib/SyTest/Federation/Server.pm

```diff
@@ -338,7 +338,6 @@ sub mk_await_request_pair
 
       return $self->{$okey}{$ikey} = Future->new
          ->on_cancel( sub {
-            warn "Cancelling unused $shortname await for @paramvalues";
```

Comment:
this was required because we now cancel the `await_request_state_ids` request if it is unused. That's normal behaviour, so we shouldn't warn on it.